src/Http/Controllers/V1/Shop/Customer/AuthController.php

```diff
@@ -93,9 +93,10 @@ public function login(Request $request)
         ]);
 
         if (! EnsureFrontendRequestsAreStateful::fromFrontend($request)) {
-            $request->validate([
-                'device_name' => 'required',
-            ]);
+            
+            if(!$request->device_name) {
+                $request->device_name = 'desktop';
+            }
```

Comment:
If the request is stateless then it's mandatory for third-party API to provide the device name. As sanctum in the backend creates a random hash on the basis of the device name.